pkg/auth/auth.go

```diff
@@ -185,6 +191,39 @@ func (s *Service) getAPIServerPort(clusterID string, ctx context.Context) (int,
 	return int(cluster.Spec.ControlPlaneEndpoint.Port), nil
 }
 
+func (s *Service) getWriteAllGroups(ctx context.Context) ([]string, error) {
```

Comment:
this function adds the groups which may be bound to `cluster-admin` only within the namespace